DependencyInjection/HttpClientExtension.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Http Client bundle.
+ *
+ * (c) David Buchmann <mail@davidbu.ch>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Http\ClientBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+/**
+ * {@inheritdoc}
+ */
```

Comment:
Author information should be added. Also, not sure if inheritdoc is needed in case of classes. If no relevant information can be given as description, the author docblock is enough.